{{cookiecutter.project_slug}}/pytest.ini

```diff
@@ -1,5 +1,7 @@
 [pytest]
 DJANGO_SETTINGS_MODULE = {{ cookiecutter.project_slug }}.test_settings
 python_files = tests.py test_*.py *_tests.py
-addopts = -p no:warnings --strict-markers --no-migrations
+addopts = --strict-markers --no-migrations
 mccabe-complexity=10
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
Pytest will no longer suppress ALL warnings in it's logs.
The `DeprecationWarning` is loud and distracting, so disabling that one.
On another project I found that some important warnings that would help you find bugs were being hidden.

This should result in the pytest error logs being a bit more useful than they were before.